app/services/location_data_service.rb

```diff
@@ -0,0 +1,60 @@
+class LocationDataService
+  # Update location tables with data from Alma
+  def self.populate
```

Comment:
Maybe call it `repopulate` to reflect the deletion that will happen? I'm a little anxious about rolling it all into one and then allowing the rake task to be invoked in production. But we'll have to run it in production at least once, right? Should we consider putting more protections on this?

I know this isn't ready for review yet and this isn't a reivew; I just was thinking about this since I saw it.

Reply:
Since this is not a common task (we'll run it once before going into production, and rarely or never at other times), maybe the answer here is not to do this in a rake task. We can open a console and run the service there when we need it.